extensions/podman/src/extension.ts

```diff
@@ -426,8 +426,13 @@ async function registerProviderFor(provider: extensionApi.Provider, machineInfo:
       await startMachine(provider, machineInfo, logger);
     },
     stop: async (context, logger): Promise<void> => {
-      await execPromise(getPodmanCli(), ['machine', 'stop', machineInfo.name], { logger });
-      provider.updateStatus('ready');
+      try {
+        await execPromise(getPodmanCli(), ['machine', 'stop', machineInfo.name], { logger });
+      } catch (err: unknown) {
+        console.error(`Error while stopping Podman machine ${err}`);
+      } finally {
+        provider.updateStatus('stopped');
```

Comment:
If it fails the status is updated to stopped. Is it correct? Wouldn't it be better to move it inside the try branch?